src/containers/homepage/survey-list.component.tsx

```diff
@@ -62,6 +84,64 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 ),
             },
         ];
-        return <Table dataSource={items} columns={columns} bordered />;
+
+        return (
+            <>
+                <Row type="flex" justify="space-between">
+                    <Col span={12}>
+                        <h2>Surveys</h2>
+                    </Col>
+                    <Col span={12}>
+                        <Link to="/survey/new">
+                            <Tooltip placement="left" title="Create a new survey">
+                                <Button type="primary" style={{ float: 'right' }}>
+                                    <Icon type="plus" />
+                                    New survey
+                                </Button>
+                            </Tooltip>
+                        </Link>
+                    </Col>
+                </Row>
+                <Table
+                    dataSource={this.filter(items)}
+                    columns={columns}
+                    bordered
+                    title={() => (
+                        <Row type="flex" justify="space-between">
+                            <Col span={16}>
+                                <span>Filter: &nbsp;</span>
+                                <RadioGroup
+                                    name="status-filter"
+                                    defaultValue="all"
+                                    onChange={(e: RadioChangeEvent) => {
+                                        this.setState({ ...this.state, statusFilter: e.target.value });
+                                    }}
+                                >
+                                    <RadioButton value="all">All</RadioButton>
+                                    <RadioButton value={SurveyStatus.Published}>
+                                        <Icon type={mapStatusToIcon(SurveyStatus.Published)} /> Published
+                                    </RadioButton>
+                                    <RadioButton value={SurveyStatus.InProgress}>
+                                        <Icon type={mapStatusToIcon(SurveyStatus.InProgress)} /> Drafts
+                                    </RadioButton>
+                                    <RadioButton value={SurveyStatus.Cancelled}>
+                                        <Icon type={mapStatusToIcon(SurveyStatus.Cancelled)} /> Cancelled
+                                    </RadioButton>
+                                </RadioGroup>
+                            </Col>
+                            <Col span={8}>
+                                <Search
+                                    style={{ float: 'right' }}
+                                    placeholder="Search surveys..."
+                                    onChange={e => this.setState({ ...this.state, search: e.target.value })}
```

Comment:
Same as above, you only need to pass fields of the state that need to be updated, no need to pass `this.state`